packages/font-icons-mdl2/src/index.ts

```diff
@@ -27,15 +27,19 @@ declare global {
   // eslint-disable-next-line @typescript-eslint/naming-convention
   interface Window {
     FabricConfig?: {
+      /**
+       * @deprecated
```

Comment:
```suggestion
       * @deprecated - Use iconBaseUrl instead.
```

Note that this message will show up on intellisense. Leaving it blank gives the user (or future maintain) no context on what to do instead.